lib/spack/spack/platforms/cray.py

```diff
@@ -36,6 +36,10 @@
 _xc_craype_dir = "/opt/cray/pe/cdt"
 
 
+def slingshot_network():
+    return os.path.exists("/lib64/libcxi.so")
```

Comment:
Ah.  Hard-coded path.  Is this guaranteed to be the path across systems?

Reply:
Prefer not having a hard-coded path BUT recognize others can be added if/when needed.